packages/source-iotsitewise/src/time-series-data/types.ts

```diff
@@ -21,6 +21,12 @@ export type AssetId = string;
 
 export type AssetModelId = string;
 
+export type RefreshRate = 1000 | 5000 | 10000 | 60000 | 300000;
```

Comment:
Nit: This should be defined in the dashboard?

Reply:
Yes good catch, its leftover from when we did not have refresh rate as a number, but a narrowed type.